libcxx/src/support/win32/thread_win32.cpp

```diff
@@ -11,9 +11,9 @@
 
 #define NOMINMAX
 #define WIN32_LEAN_AND_MEAN
-#include <windows.h>
-#include <process.h>
 #include <fibersapi.h>
+#include <process.h>
+#include <windows.h>
```

Comment:
These headers must not be reordered, otherwise we'll break the Windows build. I think I broke it when I landed the original reformatting PR. I guess we should turn off clang-format around this block.

Reply:
In Chromium we can generally convince clang-format to leave blocks in order relative to each other by inserting a newline:
```
#include <windows.h>

#include other windows headers here
```
Does that work here?

Reply:
Oh, I'm not suggesting we merge this. 

I just wanted to show the fallout from the upgrade.